values/harbor/harbor.gotmpl

```diff
@@ -149,6 +149,19 @@ notary:
       tag: {{ $tag }}
   enabled: false
   secretName: {{ $notarySecretName }}
+
+trivy:
+  resources:
```

Comment:
since you provide defaults it is enough to write:
```
{{- $h.resources.trivy }}
```

Reply:
changes made, please resolve if satisfactory